app/admin/scholarship_application.rb

```diff
@@ -0,0 +1,35 @@
+ActiveAdmin.register ScholarshipApplication do
+  permit_params :reason, :terms_accepted, :scholarship_id, :user_id
+  index do
+    selectable_column
+    id_column
+
+    column 'Scholarship' do |scholarship_app|
+      scholarship_app.scholarship.name
+    end
+
+    column 'User' do |user_id|
+      user_id.user.email
+    end
+
+    column :reason
+    column :terms_accepted
+    column :scholarship_id
+    column :user_id
+    column :created_at
+    column :updated_at
+
+    actions
+  end
+
+  form do |f|
+    f.inputs do
+      f.input :scholarship_id, label: 'Scholarship', as: :select, collection: Scholarship.all.order(:name).map { |scholarship| [scholarship.name, scholarship.id] }, include_blank: false
+      f.input :user_id, label: 'User', as: :select, collection: User.all.order(:email).map { |user| [user.email, user.id] }, include_blank: false
```

Comment:
Here we can:

- `order` by `last_name`
- use the new `full_name` method in place of `email` in the array

That way the dropdown will be sorted by last name (there are thousands of users), and the dropdown itself will contain the user's full name.